pacai/bin/pacman.py

```diff
@@ -700,7 +700,7 @@ def main(argv):
 
   argv already has the executable stripped.
   """
-
+  logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.DEBUG)
```

Comment:
Why is basic logging level here DEBUG (it should be INFO), but again should be able to be changed depending on user input.